test/e2e/support/utils/login.js

```diff
@@ -5,6 +5,12 @@
 import { WP_USERNAME, WP_PASSWORD } from './config';
 import { pressWithModifier } from './press-with-modifier';
 
+/**
+ * Perform log in with specified username and password.
+ *
+ * @param {string} username String to be used as user credential.
```

Comment:
Both `username` and `password` are optional so it should be `{?string}`.

Reply:
Updated, thank you.